lib/Makefile

```diff
@@ -55,9 +79,8 @@ $(BUILD):
 
 shader:
 	@echo "[VPCOMP] $(notdir $<)"
-	@$(VPCOMP) shader/vshader_text_normal.vcg shader/vshader_text_normal.rvp
-	@$(RAW2H)  shader/vshader_text_normal.rvp source/vshader_text_normal.vcg.h source/vshader_text_normal.vcg.S vshader_text_normal_bin
-
+	@vpcomp shader/vshader_text_normal.vcg shader/vshader_text_normal.rvp
+	@raw2h.exe  shader/vshader_text_normal.rvp source/vshader_text_normal.vcg.h source/vshader_text_normal.vcg.S vshader_text_normal_bin
```

Comment:
⚠️  this change using `raw2h.exe` directly won't work in systems that are not Windows

It's better to keep `@$(VPCOMP)` and `@$(RAW2H)` , so the user can define an export to the local tool independently of the operating system. For example:
```
export RAW2H=/my/path/to/raw2h
```

Reply:
Fixed with 7e4080cf2d40b4c5b02d3c996a457fbcdad7a8fe